README.md

```diff
@@ -107,7 +107,7 @@ Date: Thu, 19 Dec 2019 19:49:11 -0000
 Hi, Myself,
 
 Your number is 17.
->>> sent message 1
+>>> message 1 not sent (dry run)
```

Comment:
I see what you're getting at here.  I don't want the user to be confused and think that a message was sent.  I also don't want them to be confused and think there was an error.  How about this?

```suggestion
>>> message 1 sent (dry run)
```

Reply:
Well, to me _"sent (dry run)"_ sounds self-contradictory and thus more confusing.
"Would be sent, if not for --dry-run" or something to that effect would be OK.
Happy to change to whatever wording you prefer.

Reply:
Not sure which wording you prefer here; kept the original for now.

Reply:
Having something that's consistent and parseable by other tools is a plus.  I like how similar `message 1` and `message 1 sent` are.  Let's stick with `>>> message 1 sent (dry run)`